mubench.reviewsite/templates/review.phtml

```diff
@@ -335,6 +397,18 @@ use MuBench\ReviewSite\Model\Experiment;
                     }
                     b.type = "hidden";
                 }
+                function onInput() {
+                    var tag_name_input = document.getElementById("tag-input").value;
+                    var suggested_tags = document.getElementById("tags").childNodes;
+                    var form = document.getElementById("tag-form");
+                    // loops over tags in the autocompletion list to submit the form when one is clicked
```

Comment:
you are right, i did not think about that. the events dont work on the datalist tag.